.ci/jobs/pull-request-precommit.yml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
This job and the previous one `pull-request-it` look almost identical. It might be premature optimization at this stage, but we could consider turning them to templates, as the only things that seems to change is `precommit`/`it` and the `description`.